docs/about/execs.md

```diff
@@ -7,6 +7,18 @@ description: ProgSoc is run by a team of dedicated students who volunteer their
 
 ProgSoc is run by a team of dedicated students who volunteer their time to make ProgSoc the best it can be. If you have any questions about ProgSoc, feel free to contact any of the executives below.
 
+## 2024 Executives
+
+* `President` Sebastian Pietschner
+* `Vice President` Oli Lenehan
+* `Treasurer` Jakob Tonnaer Lewis
+* `Secretary` Mitchell Kijurina
+* `Marketing` Vishwas Parpattegar
+* `Events`
+ * Audrey Santoso
```

Comment:
Is mkdocs happy with 1 space for indents? Github refuses to indent anything less than four I think.

Reply:
Let me double check that.

Reply:
Fixed